### ISSUE TITLE
Most/all of the description strings should be moved under a11y

### ISSUE THREAD

**Issue by samreid** [phetsims/membrane-transport#177]

**Comment by jessegreenberg** [comment]
It seems that `listFluentParams` isn't working for terms under the `a11y` key, Ill take a look.

![Image](https://github.com/user-attachments/assets/ef9461c5-d76a-40eb-aa15-fdd519f0cff4)

![Image](https://github.com/user-attachments/assets/982dfe14-3bc6-404f-bb0d-852dd6408ad0)

**Comment by jessegreenberg** [comment]
This is happening because `convertJsonToFluent` creates new keys for the fluent terms. 

In the fluent pattern we have `{ transportProteinBriefName }`.

But in the yaml it is under 
```yaml
a11y:
  transportProteinBriefName: ...
```

Which convertJsonToFluent converts to `a11y_transportProteinBriefName` in the generated fluent. So we need to transform all references in the generated fluent, or the fluent pattern needs to handle this. I am not sure which is best.

**Comment by jessegreenberg** [comment]
Actually, it looks like some patterns are already doing this:
```
    playAreaSummaryScreen1:               "{ a11y_summary_playAreaSummaryIntro } { a11y_summary_playAreaSummarySolutes
        } { a11y_summary_playAreaSummaryBarCharts }"
    playAreaSummaryScreen2and4:           "{ a11y_summary_playAreaSummaryIntro } { a11y_summary_playAreaSummarySolutes
        } { a11y_summary_playAreaSummaryProteins } { a11y_summary_playAreaSummaryPotential
        } { a11y_summary_playAreaSummaryBarCharts }"
    playAreaSummaryScreen3:               "{ a11y_summary_playAreaSummaryIntro } { a11y_summary_playAreaSummarySolutes
        } { a11y_summary_playAreaSummaryProteins } { a11y_summary_playAreaSummaryBarCharts
        }"
```

**Comment by jessegreenberg** [comment]
This was done in the new-group-select branch and cherry-picked into fluent-json-chipper-1588 branch. @samreid do you have any thoughts on https://github.com/phetsims/membrane-transport/issues/177#issuecomment-2881763352?

In retrospect, it seems reasonable to manually have the nesting reflected in the references. I just wouldn't have known to use an underscore without inspecting things.

**Comment by samreid** [comment]
Yes, since fluent doesn't support dots in the keys we had to choose an alternative. I also think it is clear to use an absolute full string key rather than a relative string key for string accesses. So I think everything is OK here, please reopen if there is more to do.
